packages/button/CHANGELOG.md

```diff
@@ -4,7 +4,13 @@
 
 ### Patch Changes
 
--   [#5363](https://github.com/adobe/spectrum-web-components/pull/5363) [`4e06533`](https://github.com/adobe/spectrum-web-components/commit/4e065332e0236757fc3a050e53747ce82ac40ed5) Thanks [@castastrophe](https://github.com/castastrophe)! - This update aims to simplify `--mod-*` access by ensuring local variants and states aren't hooking into those custom properties for overrides. This updates all local variants and states to override the `--spectrum-button-*` properties instead and adjusts the specificity to ensure no regressions in rendered results.
+-   [#5363](https://github.com/adobe/spectrum-web-components/pull/5363) [`4e06533`](https://github.com/adobe/spectrum-web-components/commit/4e065332e0236757fc3a050e53747ce82ac40ed5) Thanks [@castastrophe](https://github.com/castastrophe)!
+
+    Adjust S1/Express static outline variant content color (from transparent-black/white to solid black/white) to fix unintentional regression.
+
+    From [@spectrum-css/button v14.1.6](https://www.npmjs.com/package/@spectrum-css/button/v/14.1.6): [#3665](https://github.com/adobe/spectrum-css/pull/3665) Thanks [@rise-erpelding](https://github.com/rise-erpelding)!
+
+    This update aims to simplify `--mod-*` access by ensuring local variants and states aren't hooking into those custom properties for overrides. This updates all local variants and states to override the `--spectrum-button-*` properties instead and adjusts the specificity to ensure no regressions in rendered results.
```

Comment:
This note about changes from 14.1.6 was meant to go in the changeset in #5363 - we overlooked it there, and that version of button has already been released, which is why this is being added to the changelog instead of a changeset.